guides/plugins/plugins/administration/handling-media.md

```diff
@@ -0,0 +1,85 @@
+# Handling media
+
+## The media upload component
+
+The Shopware 6 Administration media upload component makes it relatively easy to upload media of various kinds such as images, videos and audio files.
+This is done through the `sw-media-upload-v2` component as seen below:
+
+```html
+<div>
+    <sw-media-upload-v2
+        uploadTag="my-upload-tag"
+        :allowMultiSelect="false"
+        variant="regular"
+        :autoUpload="true"
+		label="My image-upload">
+    </sw-media-upload-v2>
+</div>
+```
+
+As you saw in the code sample, the `sw-media-upload-v2` is pretty configurable trough props.
```

Comment:
```suggestion
As you can see in the code sample below, the `sw-media-upload-v2` is pretty configurable through props.
```

just my taste

Reply:
Should we use the word `properties`, as long version?